alphastats/gui/pages/03_⚫ Preprocessing.py

```diff
@@ -0,0 +1,52 @@
+import streamlit as st
+from alphastats.gui.utils.ui_helper import sidebar_info
+
+
+def preprocessing():
+
+    with st.form("preprocessing"):
+        dataset = st.session_state["dataset"]
+
+        remove_contaminations = st.selectbox(
+            f"Remove contaminations annotated in {dataset.filter_columns}",
+            options=[True, False],
+        )
+
+        subset = st.selectbox(
+            "Subset data so it matches with metadata. Remove miscellanous samples in rawdata.",
+            options=[True, False],
+        )
+
+        normalization = st.selectbox(
+            "Normalization", options=[None, "zscore", "quantile", "linear"]
+        )
+
+        imputation = st.selectbox(
+            "Imputation", options=[None, "mean", "median", "knn", "randomforest"]
+        )
+
+        submitted = st.form_submit_button("Submit")
+
+    if submitted:
+        st.session_state.dataset.preprocess(
+            remove_contaminations=remove_contaminations,
+            subset=subset,
+            normalization=normalization,
+            imputation=imputation,
+        )
+        st.write("Data processed.")
```

Comment:
`Data processed` has little information. How about redirecting the log here or displaying a summary of what was done?